app/assets/javascripts/tylium/pages/nodes/new_form.js

```diff
@@ -17,11 +17,11 @@
 
     if ($('[data-behavior~=copy-node-label]').length) {
       $('[data-behavior~=copy-node-label]').click(function() {
-        var $modal = $(this).parents('.modal'),
-            $nodeType = $modal.find('#node_type_id'),
-            $nodesType = $modal.find('#nodes_type_id'),
-            $multi = $modal.find('#nodes_list'),
-            $label = $modal.find('#node_label');
+        var $modal = $(this).parents('[data-id~=add-node]'),
+            $nodeType = $modal.find('[data-id~=node-icon]'),
+            $nodesType = $modal.find('[data-id~=nodes-icon]'),
+            $multi = $modal.find('[data-id~=nodes-list]'),
+            $label = $modal.find('[data-id~=node-label]');
```

Comment:
Is `data-id` the best way to go here? Are we defining behaviors with these or are they really _just_ id's. If they're not behaviours or ID's maybe they're a class?

but `data-id` seems like a red flag. We're working around distinct id's by creating our own system without properly identifying why we need it.

Reply:
That's a copy-paste fail. It should be data-behavior.